launch-qemu.sh

```diff
@@ -154,8 +159,10 @@ while [ -n "$1" ]; do
 	shift
 done
 
-TMP="$EXEC_PATH/bin/qemu-system-x86_64"
```

Comment:
move these 2 lines to  where you do "QEMU_EXE=" and drop "if [ -z "QEMU_EXE" ]..." may be?
And change usage() to print $QEMU_EXE.